packages/runtime/container-runtime/src/summary/summarizerNode/summarizerNode.ts

```diff
@@ -192,6 +192,46 @@ export class SummarizerNode implements IRootSummarizerNode {
 		return { summary: result.summary, stats: result.stats };
 	}
 
+	public validateSummary() {
+		this.validateSummaryCore(false);
+	}
+
+	protected validateSummaryCore(parentSkipRecursion: boolean) {
+		assert(
+			this.wipSummaryLogger !== undefined,
+			0x1a3 /* "wipSummaryLogger should have been set in startSummary or ctor" */,
+		);
+		assert(this.wipReferenceSequenceNumber !== undefined, 0x1a4 /* "Not tracking a summary" */);
```

Comment:
Will the assert tracker dedupe this assert code?

Reply:
no, it will fail. I will update these.